privacy-config/privacy-config-api/build.gradle

```diff
@@ -28,7 +28,7 @@ java {
 
 dependencies {
 
-    api project(path: ':feature-toggles-api')
```

Comment:
I think this is actually a valid case depending on how you look at it. The api returns/uses/exposes a class from feature-toggles-api. So if we don't expose the :feature-toggles-api as an api project, then any module that would need to depend on privacy-config-api needs to automatically add the feature-toggles-api as a dependency as well.

Not 100% what is the better approach here.